lighthouse/blueprints/plates.py

```diff
@@ -81,3 +83,36 @@ def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
     except Exception as e:
         logger.exception(e)
         return {"errors": [type(e).__name__]}, HTTPStatus.INTERNAL_SERVER_ERROR
+
+
+def format_plate(barcode: str) -> Dict[str, Any]:
+    """Used by flask route /plates to format each plate
+    Arguments:
+        barcode
+    Returns:
+        {}, HTTPStatus
+    """
+    plate_map = has_sample_data(barcode)
+    number_of_positives = count_positive_samples(barcode) if plate_map else None
+
+    return {
+        "plate_barcode": barcode,
+        "plate_map": plate_map,
+        "number_of_positives": number_of_positives,
+    }
+
+
+@bp.route("/plates", methods=["GET"])
+def find_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
+    """A Flask route which returns information about a list of plates as
+    specified in the barcodes parameters.
+    This endpoint should be json and the body should be in the format
+    {"data":{"plates":[{"barcode":"12345","plate_map":true,"number_of_positives":0}]}}
```

Comment:
This comment reads a little weirdly to me - it sounds like an instruction ("should be...") but I think you are actually telling us what the return value will be? 
Would sound more natural to me to say "This endpoint will return json in the format:"

Reply:
Does the "data" part get automatically added by Flask?

Reply:
I agree that it reads weird, I was trying to follow the same pattern used in documenting the other endpoints. Happy to change is.

And no, the data bit is just me changing the returned response, and not updating the comment. I forgot that axios wrapped its responses in 'data' so saw that used on the other endpoints and was trying to follow convention. I'll update the comment.

Reply:
Could also add to this comment with instructions as to how to call the endpoint (how to pass the 'barcodes' parameter especially)